Doc/library/time.rst

```diff
@@ -708,6 +626,101 @@ The module defines the following functions and data items:
       ('EET', 'EEST')
 
 
+Clock ID Constants
```

Comment:
> It seems that settime only works for CLOCK_REALTIME though, so maybe it shouldn't link to the whole section?

Good catch. I'd keep it in the same section, but add something like ":data:\`CLOCK_REALTIME\` is the only acceptable constant for \*clk_id\*." to time.clock_settime() documentation. My wording is probably a bit bad, feel free to tweak it :)

> Also, I did not add any references in time.monotonic(), time.perf_counter(), or time.process_time(), but those seem to be shortcuts to time.clock_gettime()?

Agreed.